vite.config.js

```diff
@@ -1,15 +1,28 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
 import envCompatible from "vite-plugin-env-compatible";
+import { nodePolyfills } from "vite-plugin-node-polyfills";
 
 export default defineConfig({
-  plugins: [react(), envCompatible()],
+  plugins: [
+    react(),
+    envCompatible(),
+    nodePolyfills({
```

Comment:
We should short comment in the file here why we added that here and link to the github comment that explains it

Reply:
t is almost complete. I still need to fix a bug I mentioned earlier with the Dagre scripts. The behavior is strange because state.nodes and state.edges are updated with the correct data, but these states end up empty.

Reply:
? so what is the state of this? Is it commented or we do not need to do it anymore?

Reply:
Ok I do not see line:
`nodePolyfills({`

anymore so i guess we can resolve this comment. If so do it please.

Reply:
@palagdan ok